### PR TITLE
Vector DB Destinations: Fix OpenAI batch size and Weaviate field name collision

### PR DESCRIPTION
**Pull request by flash1293**
This PR fixes two problems:
* Rolling out https://github.com/airbytehq/airbyte/pull/31067
* Disambiguating fields properly in Weaviate to not cause field names that are incompatible with GraphQL (as it's commonly used in Weaviate)

### CODE REVIEW

**Comment by airbyte-oss-build-runner** [comment]
##  destination-qdrant test report (commit [`01c3e28eff`](https://github.com/airbytehq/airbyte/pull/31075/commits/01c3e28effd546edb08a89990efd2207973b60ff)) - ✅

⏲️  Total pipeline duration: 04mn34s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build destination-qdrant docker image for platform(s) linux/x86_64 | ✅        |
| Unit tests                                                         | ✅        |
| Code format checks                                                 | ✅        |
| Validate metadata for destination-qdrant                           | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/flash1293_fix-openai-batch-size-connectors/1696609320/01c3e28effd546edb08a89990efd2207973b60ff/destination-qdrant/0.0.4/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:01c3e28effd546edb08a89990efd2207973b60ff)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-qdrant test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-chroma test report (commit [`01c3e28eff`](https://github.com/airbytehq/airbyte/pull/31075/commits/01c3e28effd546edb08a89990efd2207973b60ff)) - ✅

⏲️  Total pipeline duration: 04mn43s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build destination-chroma docker image for platform(s) linux/x86_64 | ✅        |
| Unit tests                                                         | ✅        |
| Code format checks                                                 | ✅        |
| Validate metadata for destination-chroma                           | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/flash1293_fix-openai-batch-size-connectors/1696609320/01c3e28effd546edb08a89990efd2207973b60ff/destination-chroma/0.0.3/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:01c3e28effd546edb08a89990efd2207973b60ff)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-chroma test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-weaviate test report (commit [`01c3e28eff`](https://github.com/airbytehq/airbyte/pull/31075/commits/01c3e28effd546edb08a89990efd2207973b60ff)) - ✅

⏲️  Total pipeline duration: 05mn29s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Connector package install                                            | ✅        |
| Build destination-weaviate docker image for platform(s) linux/x86_64 | ✅        |
| Unit tests                                                           | ✅        |
| Integration tests                                                    | ✅        |
| Acceptance tests                                                     | ✅        |
| Code format checks                                                   | ✅        |
| Validate metadata for destination-weaviate                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/flash1293_fix-openai-batch-size-connectors/1696609320/01c3e28effd546edb08a89990efd2207973b60ff/destination-weaviate/0.2.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:01c3e28effd546edb08a89990efd2207973b60ff)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-weaviate test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-milvus test report (commit [`01c3e28eff`](https://github.com/airbytehq/airbyte/pull/31075/commits/01c3e28effd546edb08a89990efd2207973b60ff)) - ✅

⏲️  Total pipeline duration: 05mn30s 

| Step                                                               | Result   |
|:-------------------------------------------------------------------|:---------|
| Connector package install                                          | ✅        |
| Build destination-milvus docker image for platform(s) linux/x86_64 | ✅        |
| Unit tests                                                         | ✅        |
| Integration tests                                                  | ✅        |
| Acceptance tests                                                   | ✅        |
| Code format checks                                                 | ✅        |
| Validate metadata for destination-milvus                           | ✅        |
| Connector version semver check                                     | ✅        |
| Connector version increment check                                  | ✅        |
| QA checks                                                          | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/flash1293_fix-openai-batch-size-connectors/1696609320/01c3e28effd546edb08a89990efd2207973b60ff/destination-milvus/0.0.4/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:01c3e28effd546edb08a89990efd2207973b60ff)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-milvus test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-pinecone test report (commit [`01c3e28eff`](https://github.com/airbytehq/airbyte/pull/31075/commits/01c3e28effd546edb08a89990efd2207973b60ff)) - ✅

⏲️  Total pipeline duration: 05mn31s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Connector package install                                            | ✅        |
| Build destination-pinecone docker image for platform(s) linux/x86_64 | ✅        |
| Unit tests                                                           | ✅        |
| Integration tests                                                    | ✅        |
| Acceptance tests                                                     | ✅        |
| Code format checks                                                   | ✅        |
| Validate metadata for destination-pinecone                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/flash1293_fix-openai-batch-size-connectors/1696609320/01c3e28effd546edb08a89990efd2207973b60ff/destination-pinecone/0.0.15/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:01c3e28effd546edb08a89990efd2207973b60ff)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-pinecone test
```

File: airbyte-integrations/connectors/destination-weaviate/destination_weaviate/indexer.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def _normalize(self, metadata: dict) -> dict:
             normalized_key = key[0].lower() + key[1:]
             # "id" and "additional" are reserved properties in Weaviate, prefix to disambiguate
             if key == "id" or key == "_id" or key == "_additional":
-                normalized_key = f"__{key}"
```
Comment (pedroslopez) [inline airbyte-integrations/connectors/destination-weaviate/destination_weaviate/indexer.py:149]: is this breaking?

**Review by pedroslopez — APPROVED** [review]

**Pull request merged**